engine/src/cmd/unit.cpp

```diff
@@ -381,7 +381,7 @@ void GameUnit< UnitType >::Draw( const Transformation &parent, const Matrix &par
     if (!cam_setup_phase) {
         // Following stuff is only needed in actual drawing phase
         if (this->cloaking > this->cloakmin) {
-            cloak = (int) (this->cloaking-interpolation_blend_factor*this->pImage->cloakrate*simulation_atom_var);
+            cloak = (int) (this->cloaking-interpolation_blend_factor*this->pImage->cloakrate * SIMULATION_ATOM /*simulation_atom_var*/ );
```

Comment:
I assume all the `/*simulation_atom_var*/` is just documentation about `SIMULATION_ATOM`

Reply:
No, it's an alternative variable with the same initial value, but one that looks like maybe it's meant to be changed -- unlike the one in ALL_CAPS.

Reply:
These are places where I was using the lowercase one in an intermediate commit.

Reply:
@stephengtuggy okay - so then let's either:
- wrap these in a #define so to switch between them, or 
- if not needed clean them up

so we don't confuse people

Reply:
I believe `simulation_atom_var` is the correct one to use in most cases, so I have made that change.